### ISSUE TITLE
Loss function suddenly become zero

### ISSUE THREAD

**Issue by AmitChaulwar** [pytorch/xla#2546]
## ❓ Questions and Help
I am training a BERT model from scratch using huggingface library. Everything runs fine only the loss function became zero suddenly. What could be the reason for this?

<img width="905" alt="Screenshot 2020-10-12 at 13 52 12" src="https://user-images.githubusercontent.com/69140048/95743580-465a1780-0c92-11eb-8a52-cc10b44fdd55.png">

**Comment by taylanbil** [comment]
Could be many things, I suggest you run w/ 1 core, insert a breakpoint when loss becomes 0, and inspect the output, target, etc.

**Comment by AmitChaulwar** [comment]
Because I set the environment variable XLA_USE_BF16=1, the loss became zero.

**Comment by taylanbil** [comment]
That should not happen, there's probably a bug somewhere. May we ask for a way to reproduce this?
